test_scripts/API/AutoCompleteList/001_ATF_SetGlobalProperties_P_AutoCompleteList_OmitedText_HAPPY_PATH.lua

```diff
@@ -0,0 +1,75 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [SetGlobalProperties] Conditions for SDL to omit "autoCompleteText" param at request to HMI
+--
+-- Description:
+-- SDL must: tranfer SetGlobalProperties_request with <autoCompleteList> param and without (omited) <autoCompleteText> param to HMI
+-- respond with <resultCode_received_from _HMI> to mobile app
+--
+-- Performed steps:
+-- 1. Register Application.
+-- 2. Mobile send RPC SetGlobalProperties with <autoCompleteList> and <autoCompleteText>
+-- 3. SDL respond without <autoCompleteText>  with <resultCode_received_from _HMI> to mobile app
+----------------------------------------------------------------------------------------
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require ('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+
+--[[ General Precondition before ATF start ]]
+config.defaultProtocolVersion = 2
+commonSteps:DeleteLogsFileAndPolicyTable()
+
+--[[ General Settings for configuration ]]
+Test = require('connecttest')
+require('cardinalities')
+require('user_modules/AppTypes')
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+function Test:AutoCompleteList_Positive()
```

Comment:
@Aleksandriys, please rename test to have more obvious title, like "Test:Both_AutoCompleteList_and_autoCompleteText_present" or
"Test:Check_SDL_omits_AutoCompleteText"